server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/anker/AnkerAddressValidator.java

```diff
@@ -0,0 +1,37 @@
+/*************************************************************************************
+ * Copyright (C) 2014-2016 GENERAL BYTES s.r.o. All rights reserved.
+ *
+ * This software may be distributed and modified under the terms of the GNU
+ * General Public License version 2 (GPL2) as published by the Free Software
+ * Foundation and appearing in the file GPL2.TXT included in the packaging of
+ * this file. Please note that GPL2 Section 2[b] requires that all works based
+ * on this software must also be made publicly available under the terms of
+ * the GPL2 ("Copyleft").
+ *
+ * Contact information
+ * -------------------
+ *
+ * GENERAL BYTES s.r.o.
+ * Web      :  http://www.generalbytes.com
+ *
+ ************************************************************************************/
+package com.generalbytes.batm.server.extensions.extra.anker;
+
+import com.generalbytes.batm.server.extensions.ICryptoAddressValidator;
+
+public class AnkerAddressValidator implements ICryptoAddressValidator {
+    @Override
+    public boolean isAddressValid(String address) {
+        return true;
+    }
+    
+    @Override
+    public boolean isPaperWalletSupported() {
+        return false;
+    }
+
+    @Override
+    public boolean mustBeBase58Address() {
+        return true;
```

Comment:
Really must be base58? Than check in isAddressValid for base58 validity. See other implementations for inspiration.